### DISCUSSION TITLE
License to use for "All Rights Granted"?

### DISCUSSION THREAD

**Discussion by monotasker** [MESH-Research/knowledge-commons-repository#92] [General]
There are quite a few CORE deposits with "All Rights Granted" as the license. This isn't a recognized license, and can create some legal problems with, e.g., arguments about an implied warranty. There are also jurisdictions (like Europe) where public domain deposits are problematic. Looking at the most permissive licenses on spdx, it looks to me like 0BSD (the BSD Zero Clause License) is the equivalent to the intent of "All Rights Granted" (https://spdx.org/licenses/0BSD.html). It explicitly grants all permissions for use (including commercial) but has an implied warranty waiver.

So I'm leaning toward using that 0BSD license for CORE deposits with "All Rights Granted". 

Other options:
- CC0, but this explicitly denies any patent licenses, so it's problematic for things like software. 
- The "unlicense" is functionally equivalent to 0BSD (https://spdx.org/licenses/Unlicense.html) and I don't see any real reason to prefer one or the other. 
- Another more recent equivalent is MIT-0 (https://spdx.org/licenses/MIT-0.html) which removes the attribution requirement from the MIT free license.

In other words, I don't see any functional difference between using 0BSD, the unlicense, or MIT-0 for records listed as "All Rights Granted". I do think we should assign one of these licenses, though, to protect the owners from unwitting legal liability. Any thoughts or objections?

@bjr70 @mikethicke @ericknappe

**Comment by monotasker** [comment]
Oh, I also have a soft spot for the "Do What the Fuck You Want To Public License" ([WTFPL](https://en.wikipedia.org/wiki/WTFPL)), but it doesn't have any warranty disclaimer.

**Reply by bjr70** [reply]
I had not heard of that one and I love it. Probably not the one we should use, though. @ericknappe do you know where the "All Rights Granted" license came from? 0BSD makes sense to me as an equivalent.

**Comment by rekmla** [comment]
"All Rights Granted" - I think that was a placeholder idea that was never
replaced with something better. Seems like you have a good migration path.

On Wed, May 24, 2023 at 9:30 AM Bonnie Russell ***@***.***>
wrote:

> I had not heard of that one and I love it. Probably not the one we should
> use, though. @ericknappe <https://github.com/ericknappe> do you know
> where the "All Rights Granted" license came from? 0BSD makes sense to me as
> an equivalent.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MESH-Research/knowledge-commons-repository/discussions/92#discussioncomment-5989955>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB7ZYVVDQXRTI7ABLKSJWS3XHYEOPANCNFSM6AAAAAAYL5HUJQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <MESH-Research/knowledge-commons-repository/repo-discussions/92/comments/5989955
> @github.com>
>


-- 
R. Eric Knappe
Head of Software Development
Modern Language Association

www.mla.org

**Reply by monotasker** [reply]
Okay, thanks Eric
